app.py

```
@@ -697,6 +698,19 @@ async def delete_llm_model(name: str) -> dict[str, str]:
         raise HTTPException(status_code=404, detail=e.message)
 
 
+@api_router.put("/llm-models/{name}/default")
+async def set_default_llm_model(name: str) -> dict[str, str]:
+    """set default llm model"""
+    try:
+        await llm_config_manager.set_default_llm_model(name)
+        return {"message": "llm model set as default successfully"}
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)